### ISSUE TITLE
Ubuntu 22.04 horrible, laggy performance

### ISSUE THREAD

**Issue by jordanss1** [microsoft/vscode#224700]
**Does this issue occur when all extensions are disabled?: Yes**

**- VS Code Version: 1.92**
 **- OS Version: Ubuntu 22.04 (Linux Mint 21.3)**

**Description:**

The performance is horrible. I can only work on it for about 5 minutes before my whole computer freezes from how bad VSCode is. I used to be on windows using the same amount of apps etc and never had this issue. I have also disabled hardware acceleration and use of GPU still VSCode is unusable.

I have also done a performance monitor through the debugger and every time I press 'save' VSCode crashes without fail, so I can't even save that.

Edit: Here is the .dmp file from the crash mentioned above
[dmp.zip](https://github.com/user-attachments/files/16482700/dmp.zip)

**Steps to Reproduce:**

1. Run two local servers on your machine, pgAdmin, and a few other apps of your choice in the background 
2. Start typing, scrolling, using extensions in VSCode etc... 
3. Watch your CPU memory usage skyrocket for no apparent reason and your computer become unresponsive

**Comment by jordanss1** [comment]
Here is the .dmp file for the crash when I am trying to save the performance monitor file in debug mode

**Comment by jordanss1** [comment]
I have solved this. For anyone on Ubuntu who encounters this frustrating issue and may be new to Linux like me, try lowering your swappiness. This controls the tendency for your computer to move processes from your RAM to the disk (disk is slower than RAM) and only tends to pick up when you are using majority of your memory. 

1. Pull up terminal and run `cat /proc/sys/vm/swappiness` which should show 60
2.  Paste this into terminal `xed admin:///etc/sysctl.conf` 
3. Scroll to the bottom and paste this line and make sure to save the file afterwards `vm.swappiness=10`  then reboot your computer

Before I encountered these issues with VSCode I had it set to 30 but 10 seems to work well with my laptop.
